portfolio_website.html

```diff
@@ -17,44 +18,193 @@
+          </p>
+    
+          <ul>
+         <li class="inline"><a
+            href="https://github.com/Kaneez98Fatima"
+            target="_blank" rel="noopener noreferrer"><i class="fa fa-github icon"></i></a></li>
+          <li class="inline"><a
+            href="https://www.linkedin.com/in/kaneez-fatima-0a86601b3"
+            target="_blank"
+            rel="noopener noreferrer "
+            ><i class="fa fa-linkedin icon"></i
           ></a></li>
-        </ul>
-        </div>
-      </section>
-</body>
-</html>
\ No newline at end of file
+         <li class="inline"> <a
+            href="https://www.instagram.com/fatima98kaneez."
+            target="_blank"
+            rel="noopener noreferrer"
+            ><i class="fa fa-instagram icon"></i></a></li>
+             <li class="inline"><a href="Mail:Kaneez98fatima@gmail.com" target="_blank" rel="noopener"
+                ><i class="fa fa-envelope icon"></i
+              ></a></li>
+            </ul>
+            </div>
+        </section>
+        <section class="work-area">
+          <h4 class="recent-work">My Recent Works </h4>
+          <div class="works">
+            <div class="work-img"></div>
+            <div class="cap">
+              <h3 class="H3-W">Multi-Post Stories</h3>
+              <p class="para-1">
+                  A daily selection of privately personalized reads; no accounts or
+                  sign-ups required. has been the industry's standard dummy text ever
+                  since the 1500s, when an unknown printer took a standard dummy text.
+                </p>
+              </div>
+              <div class="flex btn-flex">
+                <ul>
+               <li><button type="button" class="cursor work-btn">css</button><li>
+                <li><button type="button" class="cursor work-btn">html</button></li>
+               <li><button type="button" class="cursor work-btn">bootstrap</button></li>
+                <li><button type="button" class="cursor work-btn">Ruby</button></li>
+              </ul>
+              </div>
+              <a href="/" class="cursor work-link">See Project</a>
+          </div>
+          <ul class="proj-flex flex">
+            <li class="proj-con">
+               <div class="proj-txt">
+                 <h3 class="proj-head">Professional Art Printing Data</h3>
+                 <p class="proj-para">
+                A daily selection of privately personalized reads; no accounts or
+                sign-ups required. has been the industry's standard dummy text ever
+                since the 1500s, when an unknown printer took a standard dummy text.
+                 </p>
+                <ul class="flex project btn-flex">
+                   <li class="proj-btn">html</li>
+                   <li class="proj-btn">bootstrap</li>
+                   <li class="proj-btn">Ruby</li>
+                </ul>
+               </div>
+               <a href="/" class="cursor project-link">See Project</a>
```

Comment:
Change the link to a button as shown in the original design below
![rigty](https://user-images.githubusercontent.com/58376020/121759035-2847fe00-cb24-11eb-8bde-9d1c4cf9af66.png)

Reply:
> Change the link to a button as shown in the original design below
> ![rigty](https://user-images.githubusercontent.com/58376020/121759035-2847fe00-cb24-11eb-8bde-9d1c4cf9af66.png)

@Shaher-11 Kindly check, according to my template, this is template 1 and my template is template 2